README.md

```diff
@@ -1,2 +1,321 @@
+    + Scan new member's ID (_e.g driver's license or passport_)
+    for Enhanced Disclosure purposes
+    (_we need to know exactly who we are working with_).
+    + Encourage them to connect with LinkedIn so people can see their skills.
+  + Ensuring _existing_ members are well looked after & needs are met.
+    + Coffee/Tea/Milk/Alternatives
+    (_ensure we are always stocked with member's favourites_)
+    + Sate of repair for all fixtures/fittings, lighting bathroom
+    + WiFi (_monthly signal and speed tests to ensure quality_)
+  + Security & Safety are regularly checked
+    + Door Entry System is checked
+    + Fire Alarm tests (_quarterly test all sensors and alarm_)
+  + Responding to _new_ member applications/registrations (_promptly_)
+  + Ensuring that people in QS have reserved their desk for the day and have
+  checked-in so we know they are in the space (_both for fire safety and billing_).
+  + Resolving any Member Disputes (_hopefully these won't happen
+  but the person we hire needs to understand that it's possible that they
+  may need to "mediate" a "so-and-so is never cleans their desk
+  and makes lots of noise..." claim ..._)
+  + Revocation of Membership
+  (_we need clear membership criteria and a "3 strikes" rule enforcement._)
++ Receiving Post/Parcels on behalf of members
+  + Opening, Scanning and Emailing post for members who want this option.
+  + Fowarding or Returning-to-Sender
+  for addressees who are no longer (_paying_) members.
++ Ensuring that all accounts for QS are up-to-date (_bills/rates paid_)
++ Periodic check for "_better deal_" on Fixed Overheads e.g: faster internet,
+better insurance or alternative (green) energy supplier.
++ Maintaining the QS website?
++ Marketing the space to potential new members?
+
+#### Remuneration
+
+We expect the Office Manager to be paid (_around_) **£9/h** (_£18k/year_) <br />
+(_i.e: not a "fortune" but well above
+["London Living Wage"](https://www.trustforlondon.org.uk/policy-change/strategic-work-item/london-living-wage/)
+and a good step up for a lot of really good
+people who are currently stuck in "Zero Hour Contract" jobs._)
+
+> _**Note**: this compensation is a "**base**" and we will gladly
+discuss "**performance related pay**" for outstanding results.
+e.g: percentage of income over full occupancy and/or additional member services_.
+
+### Suggestion Improvements (_for discussion_)
+
++ [ ] Leave door to F&C and QuietSpace open during the day so _both_ Toilets
+can be used by members.
++ [ ] A Green lightbulb outside the bathroom linked to the bathroom light so people
+can see from a glace if it's free. This can bee hooked up using IoT
++ [ ] Oil diffusers (_pleasant smell_)
++ [ ] Whiteboards (_fixed to the wall?_)
++ [ ] "Free" Fresh Fruit?
++ [ ] Bring a friend (_first day is free but they need to register
+  with valid email address/phone number for insurace purposes_)
++ [ ] "Sign-up your friends as memebers" _10% discount_ applied to both members?
```

Comment:
How long would this 10% discount be for? Ie. a month? This sounds almost infinite at the current wording. Also would there be a limit if it was infinite e.g. 10% discount can only be applied once per member?